### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8628,6 +8628,7 @@ deslects->deselects
 desltop->desktop
 desltops->desktops
 desn't->doesn't
+desnse->dense
```
Comment (sebweb3r) [inline codespell_lib/data/dictionary.txt:8631]:
```suggestion
desne->dense
desnse->dense
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -8923,6 +8925,7 @@ dicarded->discarded
 dicarding->discarding
 dicards->discards
 dicates->dictates
+dicationary->dictionary
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:8928]:
```suggestion
dicationaries->dictionaries
dicationary->dictionary
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -25575,6 +25578,7 @@ strenous->strenuous
 strentgh->strength
 strenth->strength
 strerrror->strerror
+striaght->straight
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:25581]:
```suggestion
striaght->straight
striaghten->straighten
striaghtens->straightens
striaghts->straights
```

**Review by peternewman — CHANGES_REQUESTED** [review]
A few more suggestions

**Review by peternewman — APPROVED** [review]

**Pull request merged**